src/mescal/util.clj

```diff
@@ -31,7 +31,7 @@
 
 (def ^:private accepted-timestamp-formats
   ["yyyy-MM-dd'T'HH:mm:ssZZ"
-   "yyyy-MM-dd'T'HH:mm:ss.SSSZZ"])
+   "yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSSZZ"])
```

Comment:
Tapis v3 timestamps can have 9 decimal digits now, and this format can still parse seconds with only millisecond precision.